### DISCUSSION TITLE
How can I downgrade RDP. py in version 0.9.0, as I would like it to be in version 0.4 of the Python 2.7 environment?

### DISCUSSION THREAD

**Discussion by yahui102** [thinkst/opencanary#294] [Feature Requests]
How can I downgrade RDP. py in version 0.9.0, as I would like it to be in version 0.4 of the Python 2.7 environment? Because rdp.py in 0.4 can record the password of the intruder's explosion, I can determine whether our password has been leaked and has been used as a dictionary I like the fact that the 0.9.0 version has upgraded to support https.py, so I am very confused. I cannot port https.py to 0.4, nor can I port rpd.py from 0.4 to 0.9.0. Please help me, what should I do

**Comment by jayjb** [comment]
Hi @yahui102,

Thanks for writing in. I'm moving this to an issue. Please check #298
